docs/DevBoxSetup.md

```diff
@@ -0,0 +1,83 @@
+# Dev Box Setup
+
+These instructions are for setting up your machine if you're working on the MSAL.NET library itself.
+
+## Windows Setup
+
+Windows is the primary development environment for the library at this time.
+VS for Mac is not able to understand and layout the project correctly.  You should be able
+to build from the command line.
+
+### Visual Studio 2019
+
+* Install or update VS 2019 (Community is sufficient but any SKU should work) with the following workloads:
+  * .Net Desktop Development
+  * Universal Windows Platform Development
+  * Mobile Development with .Net
+  * .Net Core cross-platform development
+
+* Then from the "Individual Components" tab, make sure these additional items are selected:
+  * .NET Framework 4.5 targeting pack
+  * .NET Framework 4.5.2 targeting pack
+  * .NET Framework 4.6.1 SDK
+  * .NET Framework 4.6.1 targeting pack
+  * .NET Framework 4.6.2 targeting pack
+  * Android SDK setup (API level 27)
+  * Windows 10 SDK (10.0.17134.0)
+
+* Android SDK level 27 (oreo) and 28 (pie), and Android SDK build tools 27.0.3 are also required. These are not installed through the VS Installer, so instead use the Android SDK Manager (Visual Studio > Tools > Android > Android SDK Manager…)
+
+### Optional but recommended tools
+
+* [Fiddler](https://www.telerik.com/fiddler)
+* [Visual Studio Code](https://code.visualstudio.com)
+
+
+## Debugging or running samples on Windows 10
+
+* The dev-built binaries are not signed so you will regularly need to run `sn -Vr *` (from an admin console) to bypass strong name validation.  If you're at MSFT on CorpNet then group policy will likely revert this setting and you may need to run this often or setup a Scheduled Task.
+* [Microsoft Only] You will need the Lab Certificate from [here](https://ms.portal.azure.com/#@microsoft.onmicrosoft.com/resource/subscriptions/57f88e5d-dc7d-422b-b87a-e215ad6a352c/resourceGroups/DevEx_Automation/providers/Microsoft.KeyVault/vaults/BuildAutomation/certificates) installed on your machines in order to access the KeyVault containing passwords for the lab accounts.
+
+## Debugging or running samples on iOS
+
+* You will need a Macintosh running the latest macOS and XCode installed
```

Comment:
I think you also need Visual Studio for Mac, as you need to have Xamarin installed. It may be that VS for Windows is able to install Xamarin remotely though, I am not sure.